lib/base.php

```diff
@@ -1047,7 +1048,27 @@ public static function handleRequest(): void {
 					OC_App::loadApps(['filesystem', 'logging']);
 					OC_App::loadApps();
 				}
-				Server::get(\OC\Route\Router::class)->match($request->getRawPathInfo());
+				$requestPath = $request->getRawPathInfo();
+				$redirects = $systemConfig->getValue('redirects', []);
+
+				if ($redirects) {
+					foreach ($redirects as $fromPattern => $toLocator) {
+						if (!preg_match('/' . $fromPattern . '/', $requestPath)) {
+							continue;
+						}
+
+						try {
+							$targetLocation = Server::get(IURLGenerator::class)->linkToRouteAbsolute($toLocator);
```

Comment:
* method `linkToRouteAbsolute` does not throw. no need in try/catch

Reply:
`get()` does